addons/website/models/website.py

```diff
@@ -512,7 +512,10 @@ def configurator_recommended_themes(self, industry_id, palette, result_nbr_max=3
     @api.model
     def configurator_skip(self):
         website = self.get_current_website()
+        theme = self.env["ir.module.module"].search([("name", "=", "theme_default")])
+        redirect_url = theme.button_choose_theme()
         website.configurator_done = True
+        return {"url": redirect_url, "website_id": website.id}
```

Comment:
What is the need of ```website_id``` to return?

Reply:
No use of in this case, just passed for the future reference & maintain the return type consistency with `configurator_appy()` method. Anyway let's remove that for now.